classes/suggested-tasks/local-tasks/providers/class-one-time.php

```diff
@@ -75,4 +205,27 @@ public function get_tasks_to_inject() {
 			$this->get_task_details( $this->get_task_id() ),
 		];
 	}
+
+	/**
+	 * Get the task details.
+	 *
+	 * @param string $task_id The task ID.
+	 *
+	 * @return array
+	 */
+	public function get_task_details( $task_id = '' ) {
+
+		return [
+			'task_id'      => $this->get_task_id(),
+			'title'        => $this->get_title(),
+			'parent'       => $this->get_parent(),
+			'priority'     => $this->get_priority(),
+			'type'         => $this->get_provider_type(),
+			'points'       => 1,
+			'url'          => $this->capability_required() ? \esc_url( $this->get_url() ) : '',
+			'description'  => '<p>' . $this->get_description() . '</p>',
```

Comment:
Tempted to move the `<p>` and `</p>` to the `get_description()`, what do you think @aristath @ilicfilip ?